### PR TITLE
Code Release

### PR DESCRIPTION
**Pull request by YiqinZhao**
Getting ready for code release.

@belindanju please review.

### CODE REVIEW

**Comment by belindanju** [comment]
@YiqinZhao Any particular things you would like me to review? Can we just do a code review during this Friday's meeting?

**Comment by YiqinZhao** [comment]
Yes, we can do it during this Friday's meeting.

File: service/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+## Directory Structure
+
+- `etc`: datasets definitions and loaders.
+- `litar`: LitAR system code.
+- `pyreality`: a 3D computation library.
+- `tests`: testing code.
+- `tmp`: a local temporary file folder (DO NOT REMOVE).
+- `config.py`: configuration file.
```
Comment (belindanju) [inline service/README.md:29]: The file name is `configs.py`.

```suggestion
- `configs.py`: configuration file.
```

File: service/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+./launch.py serve
+```
+
+## Directory Structure
+
+- `etc`: datasets definitions and loaders.
+- `litar`: LitAR system code.
+- `pyreality`: a 3D computation library.
+- `tests`: testing code.
+- `tmp`: a local temporary file folder (DO NOT REMOVE).
```
Comment (belindanju) [inline service/README.md]:
```suggestion
- `tmp`: a local temporary file folder (**DO NOT REMOVE**).
```

File: service/configs.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# System configurations
```
Comment (belindanju) [inline service/configs.py]: The quality setting is mutually exclusive. For future projects, we can consider using configuration files or flags to remove the need for manual commenting. 

```suggestion
# To use a different quality setting, you need to comment out the current setting, and then uncomment the desired setting.
# System configurations
```

File: service/litar/pipeline/dpcp/panorama/kernel.py [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+
+@cuda.jit()
+def draw_far_field(out_canvas_norms, in_anchor_acc, in_anchor_xyz, in_anchor_rgb):
+    u, v = cuda.grid(2)
+
+    n = out_canvas_norms[v, u]
+    w, r, g, b = 0, 0, 0, 0
+
+    # Should NOT do this, don't know why
```
Comment (belindanju) [inline service/litar/pipeline/dpcp/panorama/kernel.py]: Should we remove this unused code block?

File: service/service/__init__.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+
+enable_pretty_logging()
+
+
+def start_service(port=PORT, debug=True):
+    """ Holds all the registered HTTP endpoints
+    """
+
+    routes = [
```
Comment (belindanju) [inline service/service/__init__.py:18]: Add some comments about what each route is used for?

File: service/README.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+- `config.py`: configuration file.
+- `launch.py`: command line entry script
+- `Pipfile`: pipenv dependency definition file
+
```
Comment (belindanju) [inline service/README.md:32]: Add a dependency graph that shows how each component in the service interacts? The dependency can help others quickly identify which code to modify.

File: client/README.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# LitAR Reference Application
+
+This repo is a reference implementation of an AR app that is built with LitAR.
+Note that we include the LitAR client APIs via the following files:
```
Comment (belindanju) [inline client/README.md]:
```suggestion
Note that we include the LitAR APIs via the following files:
```

File: client/README.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+Then, update the `Endpoint` constant (in `Assets/Scripts/LitAR/Network/ARLightingReconstructionManager.cs`) to point to the server address. Below is an example when we set the `Endpoint` to point to our lab's server.
+
+```csharp
+private const string Endpoint = "http://cake-graphics.dyn.wpi.edu:8550";
+```
+
+To setup the LitAR reference application, please first download the Unity3D and load `client` directory as a project.
+Once the project finishes loading, use the Play button on the center top of Unity editor to start the AR application.
+
+After the application starts, please follow the Unity editor's console log information to identify the appropriate Unity's application persistent data path. For example, below is a screenshot that shows the configured path in our MacBook.
```
Comment (belindanju) [inline client/README.md]: No screenshot?

File: client/README.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+## iOS Deployment
+
+To deploy the application to iOS device, first change the project build settings to `iOS` to generate an iOS project.
+Then, compile and deploy the code to an iOS device.
```
Comment (belindanju) [inline client/README.md]:
```suggestion
Then, compile and deploy the project to an iOS device.
```

**Review by belindanju — DISMISSED** [review]

File: service/README.md (hunk 6 above)
Comment (YiqinZhao) [reply service/README.md:32]: Using this pipfile should be enough for understanding the package dependencies. I don't see noticeable benifits of describing the dependencies in a graph format.

**Pull request merged**